llvm/lib/Target/X86/X86FrameLowering.cpp

```diff
@@ -2431,7 +2431,8 @@ void X86FrameLowering::emitEpilogue(MachineFunction &MF,
   uint64_t NumBytes = 0;
 
   bool NeedsDwarfCFI = (!MF.getTarget().getTargetTriple().isOSDarwin() &&
-                        !MF.getTarget().getTargetTriple().isOSWindows()) &&
+                        !MF.getTarget().getTargetTriple().isOSWindows() &&
```

Comment:
Needs a test. You might check whether there is a test that covers isOSDarwin/isOSWindows

Reply:
Ok! I'll look. Thank you.

Reply:
Just FYI there are a couple of e2e tests that cover this change. One under review: https://github.com/llvm/llvm-project/pull/138362/files#diff-e47f971f8f461b56fd6688777c1e931fa02aa2e27cb6002f8575a1f9baa2dc26

Another one landed: https://github.com/llvm/llvm-project/pull/138361/files#diff-794c27548ad64f38994f367a0552e6b83511f1861924bd2cf42083aae690c569

But I'll continue to look if I can add a more targeted test.